DataFormats/TotemReco/src/classes_def.xml

```diff
@@ -1,10 +1,18 @@
 <lcgdict>
-  <class name="TotemT2Digi" ClassVersion="10">
-    <version ClassVersion="10" checksum="4024006040"/>
+  <class name="TotemT2Digi" ClassVersion="3">
```

Comment:
Given no single dataset was created with this extra `TotemT2Digi` object, I thought it would be a trivial change.

Reply:
> Given no single dataset was created with this extra `TotemT2Digi` object, I thought it would be a trivial change.

Yes, quite probably you are right. But let me wait for an answer from the experts of the framework

Reply:
If you are sure that there are no data with the class version 10 (that should ever be read after this change), changing the class version is acceptable (even if the change would otherwise look suspicious). The value `3` leads to a little bit smaller memory usage than `10`, so the change has some positive impact.